.ldrelease/config.yml

```diff
@@ -7,16 +9,15 @@ publications:
   - url: https://www.rubydoc.info/gems/ld-eventsource
     description: documentation
 
-releasableBranches:
+branches:
   - name: master
     description: 2.x - based on the http gem
   - name: 1.x
     description: 1.x - based on the socketry gem
 
-circleci:
-  linux:
-    image: cimg/ruby:2.5
-    context: org-global
-
-template:
-  name: ruby
+jobs:
+  - docker:
+      image: ruby:2.5-buster
+      copyGitHistory: true  # building gem requires git metadata due to use of "git ls-files" in the gemspec
```

Comment:
Is this something that should be copied over to releaser's `ruby` template instead of defined in each of the release-able Ruby project configs?

I'm comparing this with https://github.com/launchdarkly/go-server-sdk/blob/v5/.ldrelease/config.yml as an example where the template takes care of details like this. I guess the question is if we use `git ls-files` in all Ruby projects or just a select few.

Reply:
I'm not sure exactly what you're suggesting. Releaser does not currently have a way for a project template to override this property; I could change the code so it does, but I feel like a much better solution would be to modify our builds so they're not using `git ls-files` unnecessarily.

Reply:
Got it -- alright, I agree with your response on that being a preferred approach. Let's roll forward with your solution here for the time being.